README.md

```diff
@@ -10,6 +10,14 @@ Its small dimensions and its low power consumption allow the implementation in b
 
 This module utilizes the [Viam Sensor Component](https://docs.viam.com/operate/reference/components/sensor/) API and acts as a wrapper around the [BME280 ESP-IDF Component](https://components.espressif.com/components/espressif/bme280).
 
+## Building
+
+Currently, this module only supports the `esp-idf` legacy I2C drivers and needs additional flags to compile. 
+
+Either:
+- add `BINDGEN_EXTRA_CLANG_ARGS="-D CONFIG_I2C_BUS_BACKWARD_CONFIG"` to the `[env]` section of `.cargo/config.toml`
```

Comment:
Is this still true if you include the relevant header in the `bindings.h` file in this repo?

Reply:
as in adding `#include "i2c_bus.h"` to bindings.h?

This is true even when:
- importing `espressif/i2c_bus` as an extra component
- binding the i2c_bus component with bindings.h

Reply:
The build system should generate an `sdkconfig.h` file with the proper definitions (all CONFIG_ ...)

Reply:
Didn't realize that. Will give it a shot